src/test/resources/metadata-manager/srs_instance_with_electronic_access.json

```diff
@@ -432,6 +452,20 @@
               }
             ]
           }
+        },
+        {
+          "999": {
+            "ind1": "f",
+            "ind2": "f",
+            "subfields": [
+              {
+                "s": "3c4ae3f3-b460-4a89-a2f9-78ce3145e4fc"
+              },
+              {
+                "i": "00000000-0000-4000-a000-000000000000"
```

Comment:
What's the significance of this?  Is there some hidden/inferred meaning here, or could this just as easily be any old UUID?

Reply:
No extra significance, this was supposed to be an instance with electronic access, all other fields have been copied from the existing file